### PR TITLE
Add support for data_base64

### PR DESCRIPTION
**Pull request by akariv**

### CODE REVIEW

File: web/components/commons/view_mixins.py [hunk 1]
```diff
@@ -33,7 +33,7 @@ def run_pipeline(self, with_permalinks=False):
         except Exception as e:
             pipeline = None
             data['report']['error_title'] = e.__class__.__name__
-            data['report']['error_message'] = e.msg
+            data['report']['error_message'] = str(e)
```
Comment (pwalsh) [inline web/components/commons/view_mixins.py]: This breaks Py2/3 compatibility. Should use `compat.str` although I note there are places that may not be possible, like in url definitions ([ref.](https://github.com/frictionlessdata/goodtables.io/blob/master/web/components/api/__init__.py#L15)) (is this one?)

Comment (akariv) [inline web/components/commons/view_mixins.py]: The reason I made this change was that some exceptions don't have a `msg` attribute, and this causes another exception (in my case it was `TypeError`). I'll use the `compat.str` here.

Comment (pwalsh) [inline web/components/commons/view_mixins.py]: yep ok.

**Comment by akariv** [comment]
@pwalsh I updated the PR

**Comment by pwalsh** [comment]
good to merge.

**Pull request merged**